PROCESSES.md

```diff
@@ -0,0 +1,11 @@
+# Processes
+
+This document outlines processes and procedures for some common tasks in the charts repository.
+
+## Adding chart maintainers
+
+Chart maintainers are defined within the chart itself.
+So the procedure for adding maintainers is to add them there.
+The pull request which does that should also update CODEOWNERS file to that the new maintner is able to approve pull requests.
```

Comment:
I'm thinking we should link to the existing /.github/CODEOWNERS file from here, so no one misses it again 🙂

Reply:
It's linked now.